tests/test_project/settings/base.py

```diff
@@ -174,3 +174,5 @@
         "django_pgviews": {"level": "INFO", "handlers": ["console"]},
     },
 }
+
+MATERIALIZED_VIEWS_SYNC_DISABLED = False
```

Comment:
This might need to have some common prefix for this and future environment variables.

Eg. `DJANGO_PGVIEWS_*`
`DJANGO_PGVIEWS_SYNC_POST_MIGRATE=True` 

The other options for projects is to group them into a dict. 

```
DJANGO_PGVIEWS_SETTINGS = {
    "sync_post_migrate": True
}
```

@mikicz might be able to determine what naming makes sense.

Reply:
Agree @sidmitra, There is an existing setting option, `MATERIALIZED_VIEWS_CHECK_SQL_CHANGED`. I aimed to keep the new setting aligned by using the same prefix for consistency.